web/src/components/workday-accordion/workday-alert/AddZeroEntryAlert.tsx

```diff
@@ -0,0 +1,20 @@
+import { Alert } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import AddZeroEntryButton from "./AddZeroEntryButton";
+import { Dayjs } from "dayjs";
+
+export type AddZeroEntryProps = {
+  date: Dayjs;
+};
+
+const AddZeroEntryAlert = ({ date }: AddZeroEntryProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Alert severity="warning" action={<AddZeroEntryButton date={date} />}>
```

Comment:
I would add `sx={{ alignItems: "center" }}` to `Alert` props to align the contents with each other given that this is going to be single line most of the time (and it doesn't look too bad to my eye even if the text gets line-wrapped).